searx/static/themes/simple/src/less/style.less

```diff
@@ -281,24 +281,12 @@ article[data-vim-selected].category-social {
     color: var(--color-result-description-highlight-font);
   }
 
-  img {
-    &.thumbnail {
-      .ltr-float-left();
-      padding-top: 0.6rem;
-      .ltr-padding-right(1rem);
-      width: 20rem;
-      height: unset; // remove heigth value that was needed for lazy loading
-    }
-
-    &.image {
-      .ltr-float-left();
-      padding-top: 0.6rem;
-      .ltr-padding-right(1rem);
-      width: 7rem;
-      max-height: 7rem;
-      object-fit: scale-down;
-      object-position: right top;
-    }
+  img.thumbnail {
+    .ltr-float-left();
+    padding-top: 0.6rem;
+    .ltr-padding-right(1rem);
+    width: 7rem;
+    height: unset; // remove heigth value that was needed for lazy loading
```

Comment:
I remember to have see very narrowed images with a long height.
Without `max-height: 7rem` (or similar), one result may "break" the whole page (break = the user has to scroll a lot to reach the next result).

`max-height: 7rem` implies `object-fit: scale-down;`. 
And `object-fit: scale-down;` implies `object-position`.

However, I don't remember why the value is `scale-down`, [`contains`](https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit#contain) seems good too.

Reply:
This hunk just removes the `img.image` CSS and leaves the `img.thumbnail` untouched .. and I tested all engines I modified in this PR without an issue.  When I tested I compared the old rendering with the new one .. rare, but if there was a difference, the new rendering was always better and more consistent in terms of layout.

Further as the existing comment already indicates, height specifications have had a negative effect on lazy load / which currently works with `img.thumbnail` (tested).  I would be reluctant to change anything about the rendering without seeing an example that illustrates that this rendering (which we've always had) is detrimental in some way.

----

> [EDIT] see my first comment on the PR.

It is not recognizable to me what the first comment was, this was the first comment that reached me via mail.

Reply:
I clearly remember some thumbnails with an strange aspect ratio like 1:5. In this case, the layout is "broken" from the user point of view.
I haven't checked the lazy loading (it surprises me that an unknown height helps the lazy loading).
But anyway... okay.